docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
         build: frontend
         restart: always
         ports:
-            - "8080:8080"
+            - "8880:8080"
```

Comment:
Now `localhost:8501` will work too?

Reply:
if you change it here AND in config/config.yml it should work (I still haven't found a way to unify these two places)